src/browser/provider/built-in/chrome/build-chrome-args.js

```diff
@@ -0,0 +1,22 @@
+export const IN_DOCKER_FLAGS = ['--no-sandbox', '--disable-dev-shm-usage'];
+
+export function buildChromeArgs ({ config, cdpPort, platformArgs, tempProfileDir, inDocker }) {
+    let chromeArgs = []
+        .concat(
+            cdpPort ? [`--remote-debugging-port=${cdpPort}`] : [],
+            !config.userProfile ? [`--user-data-dir=${tempProfileDir.path}`] : [],
+            config.headless ? ['--headless'] : [],
+            config.userArgs ? [config.userArgs] : [],
+            platformArgs ? [platformArgs] : []
+        )
+        .join(' ');
```

Comment:
I would like to convert `chromeArgs` to `string` in the latest moment after the docker args will be added. I think it easier to find duplicates in array than in string.

Reply:
No. In this case you need to perform conditional search in `userArgs` - https://github.com/DevExpress/testcafe/pull/3557/files#diff-5fba91c54eb5e099f55ea9551c4e5686R9.
That can contain one of necessary flags also.
```js
'chrome: --no-sandbox'
'chorme: --disable-blink-features --no-sandbox --disable-file-system'
```

Reply:
Ok